tests/shared/src/main/scala/munit/Issue179FrameworkSuite.scala

```diff
@@ -13,8 +13,8 @@ object Issue179FrameworkSuite
          |5:}
          |diff assertion failed
          |=> Obtained
-         |    '''|
```

Comment:
Why the change, I think should stay, since it was there to easily copy or see what is going on

Reply:
i can provide a custom implicit instance of `DiffOptions` within this test which would preserve this. see comment above, about inconsistency of `printObtainedAsStripMargin`.